### PR TITLE
Perf yaml

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
Add description for `perf`  plugin.

##### Test Plan

1. Take a look in CI and grammar.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
Describe the PR affects users: 
- Which area of Netdata is affected by the change? documentation, integration and dashboard.
- Can they see the change or is it an under the hood? If they can see it, where? no
- How is the user impacted by the change? a plugin well documented.
- What are there any benefits of the change? A more clear information for our users.
</details>

### CODE REVIEW

**Review by sashwathn — APPROVED** [review]

**Comment by thiagoftsm** [comment]
I am setting as draft while CI does not finish analysis.

**Comment by andrewm4894** [comment]
add this to the top of the file to suppress the annoying line len yaml

```
# yamllint disable rule:line-length
---
```

**Comment by thiagoftsm** [comment]
> add this to the top of the file to suppress the annoying line len yaml
> 
> ```
> # yamllint disable rule:line-length
> ---
> ```

Done with last commit, thank you for the tip.

File: collectors/perf.plugin/metadata.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# yamllint disable rule:line-length
```
Comment (andrewm4894) [inline collectors/perf.plugin/metadata.yaml:1]:
```suggestion
# yamllint disable rule:line-length
---
```

Comment (andrewm4894) [reply collectors/perf.plugin/metadata.yaml:1]: think it wants the `---` line too as some tools look for that

File: collectors/perf.plugin/metadata.yaml [hunk 2]
```diff
@@ -1,54 +1,87 @@
...
+    method_description: "It uses syscall (2) to open a file descriptior to monitor the perf events."
   supported_platforms:
-    include: []
+    include:
+      - Linux
     exclude: []
   multi_instance: true
   additional_permissions:
-    description: ''
+    description: "It needs setuid."
```
Comment (Ancairon) [inline collectors/perf.plugin/metadata.yaml:33]: Again, I am not sure we explain if this happens by default or if there should be another action that the user needs to take.

Comment (thiagoftsm) [reply collectors/perf.plugin/metadata.yaml]: Last commits address this.

**Review by sashwathn — APPROVED** [review]

**Review by shyamvalsan — APPROVED** [review]

**Pull request merged**